static/js/runtime-config-listeners/analytics.js

```diff
@@ -0,0 +1,13 @@
+import { canonicalizeBoolean } from '../utils';
+
+export default {
+  key: 'analyticsEventsEnabled',
+  callback: value => {
+    window.AnswersExperience.AnswersInitializedPromise
```

Comment:
is there some way we could get rid of having to specify waiting for ANSWERS.init to finish? I was thinking we could add a registerListener method to AnswersExperience that would automatically wait for AnswersInitializedPromise before executing callbacks, but I'm there's probably a myriad of ways to do it
We could even add a .catch() for the AnswersInitializedPromise there, before .then()-ing the callbacks?

Reply:
I'm going to try setting up some processing that can wait for the AnswersInitializedPromise and that canonicalizes the params

Reply:
I moved canonicalization to the runtime config itself

Reply:
The waiting for the Answers intit is now setup in the AnswersExperience